Jenkinsfile

```
@@ -1 +1,5 @@
-buildPlugin(useAci: true)
+buildPlugin(configurations: [
+        [ platform: "linux", jdk: "8", jenkins: null ],
+        [ platform: "linux", jdk: "11", jenkins: null ],
+        [ platform: "windows", jdk: "8", jenkins: null, javaLevel: "8" ],
```

```suggestion
        [ platform: "linux", jdk: "8" ],
        [ platform: "linux", jdk: "11" ],
        [ platform: "windows", jdk: "8" ],
```

`javaLevel` is deprecated, and `jenkins: null` is a no-op.